app/templates/search/_categories_wrapper.html

```diff
@@ -1,5 +1,5 @@
 <div id="js-dm-live-search-categories" class="js-dm-live-search-fade">
-  <div class="lot-filters">
+  <div class="app-lot-filter">
```

Comment:
Thank you for adding `app-` to all these classes, makes my life easier when counting outstanding work 🙇